### PR TITLE
An 2137 new model build tc

### PR DESCRIPTION
**Pull request by tarikceric**
Silver and Core models

### CODE REVIEW

File: models/core/core__fact_unstake_events.yml [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        description: "{{ doc('asset') }}"
+      - name: ASSET_E8
+        description: "The asset amount for this bond, using the price table we can calculate the rune amount by asset amount"
+        tests:
+          - not_null
+      - name: EMIT_ASSET_E8
+        description: ""
+        tests:
+          - not_null
+      - name: EMIT_RUN_E8
```
Comment (eric-laurello) [inline models/core/core__fact_unstake_events.yml:47]: typo here causing an error: should be RUNE not RUN

File: models/silver/silver__update_node_account_status_events.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+version: 2
+models:
+  - name: silver__update_node_account_status_events
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - NODE_ADDRESS
+            - CURRENT_FLAG
+            - FORMER
```
Comment (eric-laurello) [inline models/silver/silver__update_node_account_status_events.yml:9]: renames in the silver caused this test to break

File: models/core/core__fact_validator_request_leave_events.yml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+      - name: BLOCK_TIMESTAMP
+        description: "{{ doc('block_timestamp') }}"
+        tests:
+          - not_null:
+              where: DIM_BLOCK_ID not in ('-1','-2')
+      - name: DIM_BLOCK_ID
+        description: "FK to DIM_BLOCK table"
+        tests:
+          - negative_one:
+              where: _inserted_timestamp <  (CURRENT_TIMESTAMP - INTERVAL '8 HOURS'
```
Comment (eric-laurello) [inline models/core/core__fact_validator_request_leave_events.yml:20]: missing a close paren at the end here

File: models/core/core__fact_unstake_events.yml (hunk 1 above)
Comment (tarikceric) [reply models/core/core__fact_unstake_events.yml:47]: Fixed

File: models/core/core__fact_validator_request_leave_events.yml (hunk 3 above)
Comment (tarikceric) [reply models/core/core__fact_validator_request_leave_events.yml:20]: Added paren

File: models/silver/silver__update_node_account_status_events.yml (hunk 2 above)
Comment (tarikceric) [reply models/silver/silver__update_node_account_status_events.yml:9]: Fixed

**Review by eric-laurello — APPROVED** [review]

**Pull request merged**